e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -652,3 +1087,11 @@ const clickHelper = buttonName => {
   cy.wait(100);
   cy.button(buttonName).click();
 };
+
+function getActionEditorModal() {
+  return cy.findByTestId("action-creator-modal");
+}
+
+function getActionExecutionModal() {
+  return cy.findByTestId("action-execution-form-modal");
```

Comment:
nit: I think we should standardize names here.

1. The component is called https://github.com/metabase/metabase/blob/9b14b870ac4f6a06baa292eca19c5ab099dea882/frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputModal.tsx#L23
2. It's `data-testid` is `action-execution-form-modal`
3. And the function here is `getActionExecutionModal`

putting these 3 names in the same case would make it more apparent that they look quite different.
1. `ActionParametersInputModal`
2. `ActionExecutionFormModal`
3. `ActionExecutionModal` (dropping `get` so we directly compare the component name)

I'm worried this discrepancy would make it hard to maintain the code.